server/chat-commands/core.ts

```diff
@@ -225,47 +229,42 @@ export const commands: Chat.ChatCommands = {
 
 		return target;
 	},
-	mehelp: [`/me [action] - Adds the given [action] into chat, attributed to the user.`],
+	mehelp: [
+ `/me [action] - Adds the user and attributes the optional [action] to them.`,
+ `/mee [message] - Like /me, but does not insert a space between the user and the message, and cannot start with a letter or number.`
+ ],
 
 	shrug(target) {
 		target = target ? ' ' + target + ' ' : '';
 		if (target.startsWith(' /me')) target = target.slice(1);
 		return this.checkChat(target + '¯\\_(ツ)_/¯');
 	},
-	shrughelp: ['/shrug [message] - Sends the given message, if any, appended with ¯\\_(ツ)_/¯'],
+	shrughelp: ['/shrug [message] - Sends ¯\\_(ツ)_/¯ appended to the optional message.'],
 
 	tableflip(target) {
 		target = target ? ' ' + target + ' ' : '';
 		if (target.startsWith(' /me')) target = target.slice(1);
 		return this.checkChat(target + '(╯°□°）╯︵ ┻━┻');
 	},
-	tablefliphelp: ['/tableflip [message] - Sends the given message, if any, appended with (╯°□°）╯︵ ┻━┻'],
+	tablefliphelp: ['/tableflip [message] - Sends (╯°□°）╯︵ ┻━┻ appended to the optional message.'],
 
 	tableunflip(target) {
 		target = target ? ' ' + target + ' ' : '';
 		if (target.startsWith(' /me')) target = target.slice(1);
 		return this.checkChat(target + '┬──┬◡ﾉ(° -°ﾉ)');
 	},
-	tableunfliphelp: ['/tableunflip [message] - Sends the given message, if any, appended with ┬──┬◡ﾉ(° -°ﾉ)'],
-
-	'battle!': 'battle',
-	battle(target, room, user, connection, cmd) {
-		if (cmd === 'battle') {
-			return this.sendReply(this.tr`What?! How are you not more excited to battle?! Try /battle! to show me you're ready.`);
-		}
-		if (!target) target = "randombattle";
-		return this.parse(`/search ${target}`);
-	},
-	battlehelp: [
-		`/battle! [format] - Starts a battle in the given [format].`,
-		`If none is given, defaults to current generation random battle.`,
-	],
+	tableunfliphelp: ['/tableunflip [message] - Sends ┬──┬◡ﾉ(° -°ﾉ) appended to the optional message'],
 
 	signout: 'logout',
 	logout(target, room, user) {
 		user.resetName();
 	},
-	logouthelp: [`/logout - Logs you out and ends your session.`],
+	logouthelp: [
+ `/logout - Logs you out and ends your session.`,
+ `You can also click the Log out button in the top-right Options gear.`,
```

Comment:
``This is equivalent to clicking the Log out button in the top-right Options gear.``

Reply:
Implemented.